thirdparty/ScrollableNotebook.py

```diff
@@ -101,7 +101,8 @@ def forget(self,tab_id):
         self.contentsManaged.pop(index)
 
     def hide(self,tab_id):
-        self.notebookContent.hide(self.__ContentTabID(tab_id))
+        #self.notebookContent.hide(self.__ContentTabID(tab_id))
+        self.notebookContent.hide(tab_id)
```

Comment:
Careful. This will be overwritten by any future updates to `ScrollableNotebook`. Not the end of the world, as I also have a set of changes I make (relating to translations) every time we pull an update.

Is this an actual bug fix to their code?